src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -92,6 +93,17 @@ const char k_MAXIMUM_NUMBER_OF_QUEUES_REACHED[] =
     "maximum number of queues reached";
 const char k_SELF_NODE_IS_STOPPING[] = "self node is stopping";
 
+const int k_MAX_INSTANT_MESSAGES = 10;
+// Maximum messages logged with throttling in a short period of time.
+
+#define LOGTHROTTLE_INFO()                                                    \
+    BALL_LOGTHROTTLE_INFO(k_MAX_INSTANT_MESSAGES, k_NS_PER_MESSAGE)           \
+        << "[THROTTLED] "
```

Comment:
There are 3 things here:

1. I believe in similar cases we made `#undef` even in `.cpp` files, when we define some custom macro
2. Might be good to place this macro definition a few lines below, so it will be under the `k_NS_PER_MESSAGE` constant which it uses
3. A small concern about `LOGTHROTTLE_INFO` being a substring of `BALL_LOGTHROTTLE_INFO`. In general, this macro name does not mention the component where it was defined, so it's a general purpose name which is not very stable when using macros

Reply:
Makes sense.  Amending